planner/core/logical_plan_builder.go

```diff
@@ -6759,6 +6759,7 @@ func (b *PlanBuilder) buildWindowFunctionFrameBound(_ context.Context, spec *ast
 	}
 
 	bound.CalcFuncs = make([]expression.Expression, len(orderByItems))
+	bound.CastFuncs = make([]expression.Expression, len(orderByItems))
```

Comment:
Maybe rename it to `compareCols` since it is not always a cast function?

Reply:
> Maybe rename it to `compareCols` since it is not always a cast function?

okk